pom.xml

```diff
@@ -301,9 +363,9 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-compiler-plugin</artifactId>
+                <version>3.8.1</version>
                 <configuration>
-                    <source>1.8</source>
-                    <target>1.8</target>
+                    <release>11</release>
```

Comment:
Are we using any 11-specific features? I know there are people who deploy this project in environments where they are not currently able to use Java beyond 8 (unfortunately..)

Reply:
I've used Guava instead where I needed default values for sets and lists. It shoudl be Java 8 ready again. (I guess it is needless to suggest alternatives such as docker and referring to https://www.oracle.com/java/technologies/java-se-support-roadmap.html#:~:text=Java%20SE%208%20has%20gone,other%20Users%20via%20java.com.)

Reply:
Yeah, as much as I'd love to move us to Java 11, it just isn't feasible for a lot of orgs around the world... Supporting a healthcare library is fun sometimes.. :)